src/Draco.Compiler/Internal/Symbols/WellKnownTypes.cs

```diff
@@ -225,7 +225,7 @@ public IEnumerable<Symbol> GetEnumEqualityMembers(TypeSymbol type)
         if (type == typeof(string)) return this.SystemString;
         if (type == typeof(object)) return this.SystemObject;
 
-        if (type == typeof(Type)) return this.SystemType;
+        if (type == typeof(System.Type)) return this.SystemType;
```

Comment:
?

Reply:
Since the IR model is also named Type, it conflict and the IDE automatically specified the namespace.

Reply:
Al, we might want to rename Type to Class, since we might have different types modeled differently in IR anyway